### PR TITLE
fix(modal): replace style tag with classname in stories

### PR DESCRIPTION
**Pull request by Rashi-Gupta-2000**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/molecules/modal/__stories__/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.Modal-list {
+    height: 185px;
+}
```
Comment (satyamyadav) [inline core/components/molecules/modal/__stories__/style.css]: @Rashi-Gupta-2000 Please add new line at EOF.

File: core/components/molecules/modal/__stories__/style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.Modal-list {
+    height: 185px;
```
Comment (satyamyadav) [inline core/components/molecules/modal/__stories__/style.css]: Please use the token instead. The nearest token to this value is `var(--spacing-8)`.

File: core/components/molecules/modal/__stories__/Dialog.story.jsx [hunk 3]
```diff
@@ -122,7 +123,7 @@ const customCode = `() => {
             description="Following 11 outreaches are currently scheduled using this sender's details."
           />
         </div>
-        <div style={{ border: 'var(--border)', height:'185px' }} className="my-5 overflow-auto">
+        <div className="my-5 overflow-auto border Modal-list">
```
Comment (satyamyadav) [inline core/components/molecules/modal/__stories__/Dialog.story.jsx:133]: Please add CSS content as a comment in the custom code to share the style with the end user. [Refer this example](https://github.com/innovaccer/design-system/blob/master/core/components/patterns/table/TableWithHeader/tableWithHeader.story.jsx#L30)

**Review by satyamyadav — CHANGES_REQUESTED** [review]

File: core/components/molecules/modal/__stories__/style.css (hunk 1 above)
Comment (Rashi-Gupta-2000) [reply core/components/molecules/modal/__stories__/style.css]: in my vscode there is an extra line of code and changes reflect if I remove it but for some reason here the extra line is not visible

**Review by satyamyadav — APPROVED** [review]

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**